### PR TITLE
Establishing new canonical site (merge on 2021-07-01)

### PR DESCRIPTION
**Pull request by mjpost**
I'll use this to setup the transition from aclweb.org/anthology → aclanthology.org, per the discussion in #1278.

See the TODOs [here](https://github.com/acl-org/acl-anthology/issues/1278#issuecomment-846417787).

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -306,22 +306,26 @@ serve:
 	 @echo "INFO     Starting a server at http://localhost:8000/"
 	 @cd build/website && python3 -m http.server 8000
 
+.PHONY: upload
```
Comment (akoehn) [inline Makefile:309]: Put the phony at the right place, right now upload nd upload-mirror are switched. (Just an optical thing)

File: Makefile [hunk 2]
```diff
@@ -306,22 +306,26 @@ serve:
...
+upload-mirror:
```
Comment (akoehn) [inline Makefile:310]: What is upload-mirror supposed to do?

**Review by akoehn — CHANGES_REQUESTED** [review]
Please add comments to what upload and upload-mirror are supposed to do.

Sry for brevity, am on mobile

File: Makefile (hunk 2 above)
Comment (mjpost) [reply Makefile:310]: We currently upload to aclweb.org (main) and aclanthology.org (mirror). Maybe I'm misunderstanding your question.

Once we've transitioned to the latter, we'll handle everything on aclweb.org with 301 redirects.

File: Makefile (hunk 1 above)
Comment (mjpost) [reply Makefile:309]: Fixed.

File: Makefile (hunk 2 above)
Comment (akoehn) [reply Makefile]: I just meant that the purpose was not clear before you added a comment.

**Comment by mjpost** [comment]
So, @akoehn, any objections to the move? I would like to do it either this week (well before NAACL starts) or after NAACL.

**Comment by nschneid** [comment]
Might be nice to wait until Zotero can catch up: zotero/translators#2380

**Comment by mjpost** [comment]
This raises an important point—there may be additional dependencies on aclweb.org/anthology. I’m going to make some announcements in different places, and we’ll set a transition date of say July 1.

Until that date, both aclanthology.org and aclweb.org/anthology are identical sites that both point to aclweb.org/anthology as the canonical site. After that date, aclweb.org/anthology will redirect everything via 301 redirects, and aclanthology.org will list itself as canonical.

**Comment by nschneid** [comment]
The latest Zotero importer is now working with aclanthology.org!

**Comment by mjpost** [comment]
Excellent, thanks for the update. We'll still wait till July 1 to merge.

**Comment by akoehn** [comment]
Todo: 
 - [ ] adjust htaccess for new host
 - [ ] move anthology-files out of the aclantholody.org dir

**Comment by mjpost** [comment]
Why the second one? That lets us have a mirror-neutral way of indicating where the files are.

**Comment by akoehn** [comment]
> Why the second one? That lets us have a mirror-neutral way of indicating where the files are.

Everything for the second point is already in place and the mirror I run has `/var/www/anthology-files` and `/var/www/html`.
The reason again it's that we want to run `rsync --delete` so that files we no longer create do not stay around.

**Comment by mjpost** [comment]
Got it. Can you share the relevant Apache config for this to save me some time? I'm not exactly sure how to split the root like this.

**Comment by akoehn** [comment]
@mjpost you probably don't need to update the apache config; the Makefile creates a symlink to the `anthology-files` directory. As long as that target is in `var/www`, a normal apache will follow that link.

I can't try that on the aclanthology.org server as I don't have root, but you can see whether
```
sudo -i
cd /var/www
mv aclanthology.org/anthology-files .
cd aclanthology.org
ln -s anthology-files ../anthology-files
```
still gives you a working system.

**Comment by mjpost** [comment]
Symlinking that way works. So the general mechanism is the Anthology creates a symlink in its root to `../anthology-files`. This seems fairly platform neutral.

But where is this in the Makefile? I don't see it.

**Comment by akoehn** [comment]
I was sure I implemented it, but I must have forgotten to do so -- will push into this branch if you don't mind.

**Comment by mjpost** [comment]
Please do!

**Comment by akoehn** [comment]
It's a bit later than I wanted but the changes should be simple enough.

**Comment by mjpost** [comment]
@akoehn I believe we're ready for the switch, when you have time to add your review.

**Review by akoehn — APPROVED** [review]
LGTM, I'm exited :-)

**Comment by mjpost** [comment]
Hold the line, there are bound to be problems that might require an emergency PR :)

**Pull request merged**